.github/workflows/orca.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       ORCA_SELF_TEST_COVERAGE_CLOVER: /home/runner/build/logs/clover.xml
       ORCA_JOB: ${{ matrix.orca-job }}
       ORCA_ENABLE_NIGHTWATCH: ${{ matrix.orca-enable-nightwatch }}
+      ORCA_COVERAGE_ENABLE: ${{ matrix.orca-coverage-enable }}
```

Comment:
I see we now have two patterns for environment variables:

- `ORCA_ENABLE_{SERVICE}`
- `ORCA_{SERVICE}_ENABLE`

We should choose one and be consistent. I leave it to you to choose which.

Reply:
Okay, I will create a new ticket address this throughout ORCA separately. Is that fine?

Reply:
Sure, that's fin.e